src/components/toasts/AddToCartToast.tsx

```diff
@@ -0,0 +1,56 @@
+"use client";
+
+import { toast } from "sonner";
+import { useRouter } from "next/navigation";
+import Image from "next/image";
+import { formatCurrency } from "@/lib/formatters";
+import { Button } from "@/components/ui/button";
+
+type AddToCartToastProps = {
+  productName: string;
+  price: number;
+  imagePath: string;
+};
+
+const AddToCartToast = ({
+  productName,
+  price,
+  imagePath,
+}: AddToCartToastProps) => {
+  const router = useRouter();
+  return (
+    <div className="flex space-x-4">
+      {/* Image container */}
+
+      <div className="flex-shrink-0">
+        <Image
+          src={imagePath}
+          width={80}
+          height={80}
+          alt="rating star"
+          className="rounded"
+        />
+      </div>
+
+      <div className="flex flex-col space-y-1">
+        <div className="font-semibold">Added to cart</div>
+        <div className="text-gray-700">{productName}</div>
```

Comment:
Let's add `line-clamp-4` in the className for them products with long fancy names

Reply:
ClassName on line 37